WordPress/src/main/java/org/wordpress/android/ui/prefs/DotComSiteSettings.java

```diff
@@ -132,6 +144,28 @@ protected void fetchRemoteData() {
         }
     }
 
+    static boolean supportsJetpackSpeedUpSettings(SiteModel site) {
+        String jetpackVersion = site.getJetpackVersion();
```

Comment:
Seems like there is an opportunity here to generalize the "check minimum Jetpack version" code now that there is this second place we need to have it (first is in `PluginUtils.isPluginFeatureAvailable()`). What do you think about introducing a helper method in `SiteUtils` to do this?

Reply:
sounds great! I'll do that